File: baekjoon_problems/math/2480.py
# 주사위 눈
a, b, c = map(int, input().split(" "))

# 세 눈이 모두 같다면
if a == b and b == c:
    print(10000 + a * 1000)
# 두 눈이 같다면
elif a == b or a == c or b == c:
    if a == b:
        print(1000 + a * 100)
    elif a == c:
        print(1000 + a * 100)
    elif b == c:
        print(1000 + b * 100)
# 세 눈이 모두 다르다면
else:
    print(max(a, b, c) * 100)

File: baekjoon_problems/math/2480.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val (a, b, c) = readLine().split(" ").map { it.toInt() }

    if(a == b && b == c) println(10000 + a * 1000)
    else if(a == b || b == c || c == a) {
        when {
            a == b -> print(1000 + a * 100)
            b == c -> print(1000 + b * 100)
            else -> print(1000 + c * 100)
        }
    }
    else print(arrayOf(a, b, c).maxOrNull()?.times(100))
}